src/features/organizations/components/ActivistPortlHeader/index.tsx

```diff
@@ -1,6 +1,7 @@
 import { Box, Button } from '@mui/material';
-import { FC, useState } from 'react';
-import { usePathname, useRouter } from 'next/navigation';
+import { FC, useMemo, useState } from 'react';
```

Comment:
Putting this comment here towards the top of the file, but it's not specifically about this line.

This file is in a misspelled directory. The name `ActivistPortlHeader` should be `ActivistPortalHeader`. Maybe this is a good opportunity to rename it?